src/Microsoft.AspNetCore.Antiforgery/Internal/AntiforgeryOptionsSetup.cs

```diff
@@ -28,7 +28,14 @@ public static void ConfigureOptions(AntiforgeryOptions options, DataProtectionOp
 
         private static string ComputeCookieName(string applicationId)
         {
-            using (var sha256 = SHA256.Create())
+            HashAlgorithm sha256;
+#if NETSTANDARD1_3
+            sha256 = SHA256.Create();
+#else
+            sha256 = new SHA256Cng();
```

Comment:
After verification, it looks like the type exists on Mono. It just uses `SHA256Managed` internally.